src/icons/GearIcon.tsx

```diff
@@ -8,7 +8,7 @@ interface Props {
 }
 
 const GearIcon = ({ size = 22, color = colors.black }: Props) => (
-  <Svg width={size} height={size} viewBox="0 0 22 22" fill="none">
```

Comment:
Why are we changing this? What's the actual viewBox which was used in the original design?

Reply:
@jeanregisser Original it was `0 0 22 22` and when it was on the left it wasn't as visible but once I moved it to the right - it appeared to be a notch off to properly horizontally align with qrcode and notification icons. Those 2 pixels basically fixed it.

Reply:
Should the height be set to 20/22 * size for this to scale properly with different sizes? alternatively, could you just replace this with a 24x24 sized svg similar to the other icons here? Think you can copy the svg from one level up in the figma

<img width="59" alt="image" src="https://github.com/user-attachments/assets/e96322a5-b5a1-4978-a702-27838c7d8b17">

Reply:
@satish-ravi of course, will do!

Reply:
@satish-ravi fixed!